handbook/product/design/metrics/index.md

```diff
@@ -1,12 +1,35 @@
 
-To define a metric, designers should use a [metric card](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/%F0%9F%9B%A0Project-Tools?node-id=2597%3A6172) from the [project tools](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/%F0%9F%9B%A0Project-Tools?node-id=0%3A1&viewport=-1298%2C3150%2C1) library. These cards help standardize the names and description of metrics across the product.
+Measurement plays a role in design because decisions we make every day can be subjective. Additionally, user testing with smaller sets of users is often not sufficient to surface issues. By setting goals and reviewing how our designs perform against those goals, we can remove some of this uncertainty and spot problems early.
 
-Follow the [guidelines for naming events](../../../ops/bizops/amplitude.md#adding-events-to-amplitude) when defining an event.
+A practical example of using discovery and measurement to improve an OKR:
 
-- If a large number of events are being collected, consider creating a Metrics page [Figma](https://www.figma.com/file/LfQUGTLvfYWxyuFoyQsptB/?node-id=251%3A7408) to identify the locations and interactions where events will be collected.
-- Metrics review: consider requesting a review of your metrics definitions from the analytics team before the project goes to development.
+> Improving retention is a key OKR. The search product team included a video in a recent homepage update. While reviewing the events correlated with high retention, it is discovered that users who view the video are more likely to retain, but that the overall views of the video (and thus retained users) are small.
+
+From this discovery, a designer could place greater importance on the video or include it in various segments of the onboarding flow to elicit more views and thus improve retention.
+
+## Analytics tooling
+
+Sourcegraph currently uses three solutions for product analytics:
+
+### Amplitude
+
+Amplitude provides session-based event data of Sourcegraph cloud features. Designers can use Amplitude for both discovery and measurement of user retention, usage metrics, funnels, path journeys, and other quantitative analysis requirements.
+
+[Learn more about using Amplitude as a designer](./amplitude-for-designers.md)
+
+### Looker
+
+Looker provides access to all event log data across both on-premise installations and cloud products from a single interface. Because Looker data is provided primarily from ‘ping’ data, it can answer cumulative questions about events but does not provide events related to individual user sessions.
+
+### Google Analytics
+
+Google Analytics provides session-based data that is useful for reviewing how users interact with the cloud product. Useful information which can be derived from Google Analytics includes frequency, technology (browser, mobile device, etc.), conversion against goals and behavior (content, flows, and events). Data is only available for users who have accepted the Sourcegraph cookies.
```

Comment:
Can we add information about how to access Google Analytics data? Is it publicly accessible? Google Analytics was never mentioned during the onboarding process as a source of usage data.

Reply:
```suggestion
Google Analytics provides session-based data that is useful for reviewing how users interact with the cloud product. Useful information which can be derived from Google Analytics includes frequency, technology (browser, mobile device, etc.), conversion against goals and behavior (content, flows, and events). Data is only available for users who have accepted the Sourcegraph cookies.

Google Analytics access can be provided by [#it-tech-ops](https://sourcegraph.slack.com/archives/C01CSS3TC75). It can be accessed at [analytics.google.com](https://analytics.google.com)
```

Reply:
@jjinnii LMK if this change fits what you are looking for! If there is designer-specific onboarding, designers should be added to that list. Can you follow up on that?

Reply:
Sure, I can add Google Analytics to [Design Tools](https://about.sourcegraph.com/handbook/product/design/tools)! There was no designer-specific tools section in ProcessStreet, so this is probably the place to do it.